docs/ble.md

```diff
@@ -25,21 +38,23 @@ treat them like decimals.*
 Web IDL
 -------
 This IDL provides an overview of the interface; see below for documentation of
-specific API functions.
+specific API functions.  Click [here](Notes_on_WebIDL.md) for an
+explanation of zephyr.js' WebIDL conventions.
+
 
 ```javascript
 // require returns a BLE object
 // var ble = require('ble');
 
-[NoInterfaceObject]
+[ReturnFromRequire]
 interface BLE: EventEmitter {
```

Comment:
And this would be

```
class BLE extends EventEmitter {
  disconnect(address: string): void;
  startAdvertising(name: string, uuids: string[], url: string): void;
  stopAdvertising(): void;
  setServices(services: PrimaryService[]) void;
  newPrimaryService(init: PrimarySeviceInit): PrimaryService;
...
```

Reply:
It is very JS like but has a bit of extra info, like types, readonly etc